Wrappers.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+#include "Synthesizer.h"
+#include "Restaurant.h"
+#include "Bank.h"
+#include "EffectsRack.h"
+#include "DiningRoom.h"
```

Comment:
None of these types are used in this header file, excluding the nested types.
Therefore, all of the types can be Forward Declared here instead, and these #include statements can be moved to the CPP file.
- [ ] revise